lib/bamboo/adapters/send_grid_helper.ex

```diff
@@ -14,6 +14,29 @@ defmodule Bamboo.SendGridHelper do
 
   @id_size 36
   @field_name :send_grid_template
+  @custom_args :custom_args
+
+  @doc """
+  Specify a custom argument values that are specific to the entire send that will be carried along with the email and its activity data.
+  Substitutions are not made on custom arguments within SendGrid.
+  The combined total size of these custom arguments may not exceed 10,000 bytes.
+  Custom args must be a `String.t`.
+
+  ## Example
+
+      email
+      |> with_custom_args("userId", "12345")
+  """
+  def with_custom_args(email, arg_name, arg_value) do
+    if is_binary(arg_name) and is_binary(arg_value) do
```

Comment:
@coryt do you think we need to enforce that the value is a string here or could we accept anything and call `to_string/1` below when we assign it?

The raise below is doing essentially that with `#{arg_name}:#{arg_value}`

Reply:
Sorry for the delay @doomspork 
This makes sense, I'll update this